CloudDataProtection/CloudDataProtection.Core/Messaging/IRpcServerBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace CloudDataProtection.Core.Messaging
+{
+    public interface IRpcServerBase<TRequest, TResponse>
```

Comment:
Hernoemen naar IRpcServer

```suggestion
    public interface IRpcServer<TRequest, TResponse>
```